src/django_components/component.py

```diff
@@ -1081,17 +1081,15 @@ def _render_impl(
         # Allow to access component input and metadata like component ID from within these hook
         with self._with_metadata(metadata):
             context_data = self.get_context_data(*args, **kwargs)
-            # TODO - enable JS and CSS vars
-            # js_data = self.get_js_data(*args, **kwargs)
-            # css_data = self.get_css_data(*args, **kwargs)
+            # TODO - enable JS and CSS vars - EXPOSE AND DOCUMENT AND MAKE NON-NULL
```

Comment:
For the tests, I wanted to check that the data from `get_js_data` and `get_css_data` do get stored too. But since that feature's not yet enabled, I did this little change so that the test components can define those methods.